packages/flutter/lib/src/material/chip.dart

```diff
@@ -1416,6 +1428,8 @@ class _RawChipState extends State<RawChip> with TickerProviderStateMixin<RawChip
   void initState() {
     assert(widget.onSelected == null || widget.onPressed == null);
     super.initState();
+    _updateState(MaterialState.disabled, !widget.isEnabled ?? false);
```

Comment:
How could !widget.isEnabled be null (and we assert that isEnabled is non-null)? Likewise for the next line and similar expressions below.

Reply:
Good call, we actually already assert `isEnabled != null`, so I removed the `?? false` here. I'm worried about doing the same thing for selected, because in our tests (and possibly users code) we do something like `selected: isSelectable ? _selected : null`.  

https://github.com/flutter/flutter/blob/975156e91c227c0c6636259c17f5800082d654a1/packages/flutter/test/material/chip_test.dart#L1171